CONTRIBUTING.md

```diff
@@ -104,31 +104,38 @@ First, you need to get ahold of the code, so you have a copy of it locally that
 
 ### Configuration
 
-* If you use a Ruby version manager such as RVM, rbenv or chruby, then: `cp .ruby-version.example .ruby-version`
-* Copy the environment config: `cp config/env .env`
-* Open `.env` and add the **Client ID** and **Client Secret** from the previous GitHub OAuth steps.
+In most cases, it is easiest to run the `bin/setup` script to automatically setup the defaults for the application. The script will:
 
-All the commented out values in `.env` can be left alone for now.
+* setup the `.ruby-version` file for your Ruby version manager such as RVM, rbenv or chruby
+* copy the environment config to `.env`
+* run bundler to install the required gems
+* create the development database using the credentials in `config/database.yml`
+* download and seed the database with a good starting set of data
+* create the test database
+* run the test suite
 
-You don't need to fill in the EXERCISES_API value unless you're going to be working on the x-api codebase.
+Then you'll need to:
 
-### Dependencies
+* Open `.env` and add the **Client ID** and **Client Secret** from the previous GitHub OAuth steps.
 
-Next, make sure all the application dependencies are installed:
+All the commented out values in `.env` can be left alone for now.
 
-* Install gems with: `bundle install`
+You don't need to fill in the EXERCISES_API value unless you're going to be working on the x-api codebase.
 
 ### Data
 
-Finally, set up the database. This means both creating the underlying database, and migrating so that it
-has all the correct tables. Also runs a script to add fake data, so there are things to click on and look at while working on the app.
+The script in `bin/setup` will creating the underlying database, and migrating so that it has all the correct tables. It also runs a script to add fake data, so there are things to click on and look at while working on the app.
 
-You can now set up the database for development:
+You can now set up the database for development manually:
 
 * Do all of it in one go: `rake db:from_scratch`
```

Comment:
It sounds like the bin/setup script will do the db from scratch thing. Shall we delete lines 129-134 as well?